### PR TITLE
Fix "Description in the doc is wrong"(#318)

### LINKED ISSUE #318: Description in the doc is wrong
**Describe the bug**
1. The "Instance Status change SpringBoot actuator endpoint" described in [doc](https://docs.spring.io/spring-cloud-zookeeper/docs/current/reference/html/#instance-status) is out of date.

After the [PR](https://github.com/spring-cloud/spring-cloud-commons/pull/701) has been merged, the "Instance Status change SpringBoot actuator endpoint" should be `serviceregistry` instead of `service-registry`.

2. According to the code in class [ConfigWatcher](https://github.com/spring-cloud/spring-cloud-zookeeper/blob/main/spring-cloud-zookeeper-config/src/main/java/org/springframework/cloud/zookeeper/config/ConfigWatcher.java#L111), for the lastest version, the configuration will be reloaded when updates are made to the ZooKeeper configuration namespace, instead of mannually call the `/refresh` actuator endpoint.

**Sample**
1. 
  image
  image
2.
  image
  image

### PR DESCRIPTION
**Pull request by Pengfei-Lu**
Fix #318

### CODE REVIEW

**Comment by Pengfei-Lu** [comment]
Any update about this?

**Comment by Pengfei-Lu** [comment]
Any update about this? @spencergibb

**Comment by Pengfei-Lu** [comment]
I've fix the conflicts with the main branch. Please check. @OlgaMaciaszek

**Review by OlgaMaciaszek — CHANGES_REQUESTED** [review]
Thanks @Pengfei-Lu . Sending a HTTP `POST` request to `/refresh` also works. Please update your change to include it.

**Comment by Pengfei-Lu** [comment]
> Thanks @Pengfei-Lu . Sending a HTTP `POST` request to `/refresh` also works. Please update your change to include it.

@OlgaMaciaszek  I've updated the change. Please check. Thanks.

File: docs/modules/ROOT/pages/config.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ only to the instances of the service named `testApp`.
 
 Configuration is currently read on startup of the application. Sending a HTTP `POST`
 request to `/refresh` causes the configuration to be reloaded. Watching the configuration
-namespace (which Zookeeper supports) is not currently implemented.
+namespace (which Zookeeper supports) is currently implemented also.
```
Comment (OlgaMaciaszek) [inline docs/modules/ROOT/pages/config.adoc:28]: Please change from "is currently implemented also." to "is also available.".

**Review by OlgaMaciaszek — CHANGES_REQUESTED** [review]
Thanks @Pengfei-Lu. I have added a comment. Please address.

File: docs/modules/ROOT/pages/config.adoc (hunk 1 above)
Comment (Pengfei-Lu) [reply docs/modules/ROOT/pages/config.adoc]: Done. Please check.

**Review by OlgaMaciaszek — APPROVED** [review]
LGTM.

**Pull request merged**